internal/action/command.go

```diff
@@ -486,13 +486,15 @@ func (h *BufPane) VSplitCmd(args []string) {
 		return
 	}
 
-	buf, err := buffer.NewBufferFromFile(args[0], buffer.BTDefault)
-	if err != nil {
-		InfoBar.Error(err)
-		return
-	}
+	for _, a := range args {
+		buf, err := buffer.NewBufferFromFile(a, buffer.BTDefault)
+		if err != nil {
+			InfoBar.Error(err)
+			return
+		}
 
-	h.VSplitBuf(buf)
+		h.VSplitBuf(buf)
+	}
 }
 
 // HSplitCmd opens a horizontal split with file given in the first argument
```

Comment:
Update this documentation?

Reply:
Legit and for consistency reason for the already present `tab` use case too.

Reply:
So, what about updating this comment?